### DISCUSSION TITLE
Agent Pool Service (1) user deleted

### DISCUSSION THREAD

**Discussion by leandroamore** [actions/virtual-environments#6900] [General questions]
Hi, we deleted a bunch of Agent Pool Service (x) users from azure DevOps and now we can't user the Microsoft-Hosted Agents any more. Is there any way to recreate this users?

**Comment by mikhailkoliada** [comment]
Hello! This question is better to be directed to https://answers.microsoft.com/en-us, this repo is for image specs used for hosted agents, it does not concern end-user ui/settings experience :)
